akaibot.py

```diff
@@ -57,22 +58,33 @@ async def on_message(self, message: discord.Message):
         await self.command.handle(message)
 
     async def on_raw_reaction_add(self, payload: discord.RawReactionActionEvent):
-        self.logger.debug('Recived reaction add info')
-        self.logger.debug(f'User {payload.member.display_name} reacted with {payload.emoji.name} '
-                         f'to message {payload.message_id}')
+        member = self.get_guild(payload.guild_id).get_member(payload.user_id)
+        self.logger.debug('Received reaction add info')
+
+        if payload.emoji.name[:-1] == "thanks":
+            points = int(payload.emoji.name[-1])
+            msg = await self.get_channel(payload.channel_id).fetch_message(payload.message_id)
+            self.helper.add_points(msg.author, member, self.user, points)
+
         if payload.message_id != int(self.settings.at_key('reaction_role_message_id')):
             self.logger.debug('Reaction message is not recognized as subscribed')
             return
```

Comment:
Czy to nadal potrzebne? Jest jakieś istotne reaction_role_message_id?

Reply:
Ja to zostawiłem na przyszłość, kiedy bedzięmy ogarniać nadanie roli projektu za pomocą reakcji.

Reply:
No to najwyzej napiszę to ponownie, teraz nam się nie przyda